docs/content/Schema/Reference/cube.mdx

```diff
@@ -735,6 +703,19 @@ It has the following structure:
 FILTER_PARAMS.<CUBE_NAME>.<FILTER_NAME>.filter(expression)
 ```
 
+<WarningBox>
+
+Heavy usage of FILTER_PARAMS is considered a bad practice. It usually leads to hard to maintain data model.
+
+Good rule of thumb is to use FILTER_PARAMS only for performance optimizations.
```

Comment:
```suggestion
Good rule of thumb is to use `FILTER_PARAMS` only for performance optimizations.
```

Reply:
Should we say specifically that its useful for predicate pushdown scenarios? "Performance optimizations" sounds kind of vague...

Reply:
yes, let's do this, can you make a change?